pam/integration-tests/vhs-helpers_test.go

```diff
@@ -35,6 +37,7 @@ type tapeData struct {
 	Name     string
 	Outputs  []string
 	Settings map[string]any
+	Env      map[string]*string
```

Comment:
Why must it be a `map[string]*string`?

Reply:
So... I wanted to still give the possibility to set an empty env variable, since `foo="" && [ -v foo ]` is different from `foo="" && [ -n "$foo" ]`, so I wanted to be able to still support it.

We can change it later if you want, but I don't see an juge problem with it.

Reply:
I don't like the idea of setting up things like this for something that __might__ happen in the future... I think we should use normal strings for now and then switch to `*string` when the need comes (and if it comes)